packages/playground/src/components/algorand_capacity.vue

```diff
@@ -86,16 +86,10 @@ watch(storage, storage => emits("update:storage", storage), { immediate: true })
 watch(
   () => [custom.value, props.network, props.type] as const,
   ([custom, network, type]) => {
```

Comment:
```suggestion
watch(
  () => [props.network, props.type] as const,
  ([network, type]) => {
```

Remove unused property